x-pack/plugins/alerting/server/plugin.ts

```diff
@@ -291,7 +291,7 @@ export class AlertingPlugin {
     savedObjects: SavedObjectsServiceStart,
     request: KibanaRequest
   ) {
-    return savedObjects.getScopedClient(request, { includedHiddenTypes: ['alert'] });
+    return savedObjects.getScopedClient(request, { includedHiddenTypes: ['alert', 'action'] });
```

Comment:
This is needed as the AlertsClient interacts with the `action` SavedObject type directly instead of using the ActionsClient. Changing this would make this PR much more complex and isn't actually needed to achieve what this PR sets out to do.

I've submitted an issue to address this: https://github.com/elastic/kibana/issues/67163